### DISCUSSION TITLE
Why am I requested to pay $44/year for GitHub Team, when I was told I would get it free, being a teacher approved for the GitHub Education program?

### DISCUSSION THREAD

**Discussion by ItaiBEdu** [community/community#65163] [GitHub Education] [answered]
I am a computer science high-school teacher. As such, I have enrolled to the GitHub education program (and have been approved). I was informed that my membership in the program makes me eligible for a free GitHub Team membership (see the following screenshot).

![GitHubTeamFree](https://github.com/community/community/assets/143313920/f030f723-efe3-4ae9-b173-488fb7e5cd9a)

I would like to enrol in GitHub Team.

After accessing [the GitHub Team homepage](https://github.com/team), and clicking the "Get started with Team" button, I was presented with a form requesting me to enter my organization name and whether the organization belongs to my personal account or to a business or institution. I chose "my personal account" in light of [the answer I received in this forum to a previous question of mine](https://github.com/orgs/community/discussions/65158#discussioncomment-6839622).

I was then directed to a page where I was informed my GitHub Team membership would cost me $44/year, and was request to enter billing details (see the following screenshot).

![GitHubTeamBilling](https://github.com/community/community/assets/143313920/7d4fb1cd-a8c7-4bd1-901b-d108eeec1fed)

Why am I required to pay and enter billing details if my GitHub Team membership is supposed to be free, as a teacher enrolled in the GitHub education program? How can I claim my free GitHub Team membership?

**Comment by arafajs** [comment, marked as answer]
I think the reason for this is that you chose to create a GitHub Team for your personal account. When you do this, you are not eligible for the free GitHub Team membership for education. Instead, you will need to pay for a regular GitHub Team membership, which costs $44/year.

To get the free GitHub Team membership, you need to create a GitHub Team for your organization. Your organization can be your school or any other educational institution that is part of the GitHub Campus Program.

**Reply by ItaiBEdu** [reply]
Oh, I see. I first have to create an organization inside my account, and then upgrade this organization to Team. Thanks!

**Reply by bmitc** [reply]
Could you give more direct instructions how you did this? I applied and approved for the free GitHub Team for education as well, but I see no option to upgrade or create a new organization for free GitHub Team. For example, I have already created a new GitHub.com organization prior to applying for the program. When I setup the program, I did not select the personal option but rather assigned it to my university institute. Now, when I go into the organization, I see no option to upgrade the organization (for free) to GitHub Team.

**Reply by bmitc** [reply]
> To get the free GitHub Team membership, you need to create a GitHub Team for your organization. Your organization can be your school or any other educational institution that is part of the GitHub Campus Program.

The "FREE GitHub Team for courses, coding clubs, and nonprofit research" program/discount is different than the [GitHub Campus program](https://education.github.com/schools). For example, GitHub Campus is GitHub Enterprise, not Team.

**Reply by rnyang** [reply]
@bmitc did you end up having success with this? I am in the same situation: approved as a teacher, but not seeing an option to upgrade.

**Reply by bmitc** [reply]
Yes, I did. See my other comment here: https://github.com/orgs/community/discussions/65163#discussioncomment-9426201. You just have to wait a few days until you get an email stating that your benefits are ready.

**Comment by hotdogontology** [comment]
I have essentially the same question, but the answer does not make any sense to me. I have applied for education benefits, I have linked my personal account, I have been approved to receive the teacher benefits, and I have made an organization for my school that I wish to upgrade to the free Teams account for education, but that option is not available. What am I misunderstanding here?

**Reply by bmitc** [reply]
What I found out is that you have to wait several days for the benefits to actually kick in, even after you have been approved. You will get an email stating that your education benefits are now available. Once they are, then you should start seeing the ability to upgrade organizations to GitHub Team for free. I want to say that it took around three to five days for me.

**Comment by AbdulAhad365** [comment]
yes,  I think the reason for this is that you chose to create a GitHub Team for your personal account. When you do this, you are not eligible for the free GitHub Team membership for education. Instead, you will need to pay for a regular GitHub Team membership, which costs $44/year.

**Reply by rnyang** [reply]
In that case, what is the correct path to be eligible for Github Team membership for education?

**Reply by bmitc** [reply]
@AbdulAhad365 I don't believe that is true. The answer to the original question is: just wait longer.

From GitHub after applying to the GitHub education benefits:

> There may be a wait period between verification and access to academic benefits.

**Comment by ProfWentao** [comment]
I have been approved as an educator a while back and was able to upgrade my existing organizations to github team. Today I created a new organization but was not able to upgrade it to github team for free. How come?

**Reply by ProfWentao** [reply]
I stumbled on the solution. Apparently github does not talk to github education in some scenarios. You can get the free upgrade to GitHub Team for all your organizations if you visit this page: [](https://education.github.com/globalcampus/teacher).

**Reply by bmitc** [reply]
I was going to respond here regarding that, but yes, that is indeed the solution that I previously discovered as well.

**Reply by Ashish-Subramanian** [reply]
Sorry, what is the page @ProfWentao? Seems to have been deleted, or otherwise I can't see what I assume was a link.

**Reply by bmitc** [reply]
Go here https://education.github.com/globalcampus/teacher and click on **Upgrade to GitHub Team**.

In case links are removed, the address is:

```
https://education.github.com/globalcampus/teacher
```

**Reply by Ashish-Subramanian** [reply]
Thank you very much!

**Reply by zaikunzhang** [reply]
> Go here [education.github.com/globalcampus/teacher](https://education.github.com/globalcampus/teacher) and click on **Upgrade to GitHub Team**.
> 
> In case links are removed, the address is:
> 
> ```
> https://education.github.com/globalcampus/teacher
> ```

This is what finally worked for me. Thank you!
